components/SignIn.js

```diff
@@ -109,12 +109,12 @@ export default class SignIn extends React.Component {
                   <Image src="/static/images/oc-logo-oauth.png" alt="Open Collective logo" height={104} width={104} />
                 </Box>
                 <Box ml={24} mr={24} mt={32} minWidth={40}>
-                  <Image src="/static/images/oauth-flow-connect.png" alt="OAuth Connect" height={40} width={40} />
+                  <Image src="/static/images/oauth-flow-connect.png" height={40} width={40} />
                 </Box>
                 <Box minWidth={104}>
                   <img
                     src={this.props.oAuthAppImage}
-                    alt="OAuth Logo"
+                    alt="Logo of open authorize application"
```

Comment:
I don't see the value of this alt in term of accessibility. Better remove it.

Reply:
What about the one just before "Open Collective logo", how is it useful for accessibility?

Reply:
I've removed that as well. I realized that we have lots of places that the `Open Collective Logo` string is used as alt strings. I've opened a new PR to remove all of them; https://github.com/opencollective/opencollective-frontend/pull/8005